resources/views/settings/settings.blade.php

```diff
@@ -0,0 +1,79 @@
+@extends('layouts.main_layout')
+
+@section('head')
+
+    <title>Settings</title>
+
+@endsection
+
+@section('mainBody')
+
+    <main class="primary-main row">
+        <section class="grid register-main">
+
+            <form class="form-horizontal">
```

Comment:
Per the sprint requirement, we do need some sort of functionality in. Having an actual setting be able to be modified and _used_ is more important than having a lot of bogus settings on this page.